config/routes.rb

```diff
@@ -399,6 +399,7 @@
           get 'resource_pools'
           post 'ping'
           put 'associate'
+          put 'invalidate_cache'
```

Comment:
nitpick: i would prefer reload vs invalidate (sounds scary as we all know cache invalidation is a hard problem ;))